assets/src/scss/components/_demoBlock.scss

```diff
@@ -33,9 +33,14 @@
         padding: spacing(x5) spacing(x5);
         position: relative;
     }
+        .demo-visual--spacingBottom {
```

Comment:
Not really part of your PR, but any idea what the `demo-visual-bottomSpacer` does?  Just did a quick search for it and it doesn't look like it's used, so may be able to get rid of it…

Reply:
It's not used, no. Neither is `demo-visual-logOutput`. I'll remove them whilst I'm here.